MantidQt/CustomInterfaces/src/Indirect/ApplyPaalmanPings.cpp

```diff
@@ -221,21 +192,18 @@ void ApplyPaalmanPings::run() {
       }
     }
 
-    // If not in wavelength then do conversion
-    std::string originalCanUnits = canClone->getAxis(0)->unit()->unitID();
-    if (originalCanUnits != "Wavelength") {
-      g_log.information("Container workspace not in wavelength, need to "
-                        "convert to continue.");
-      absCorProps["CanWorkspace"] = addConvertUnitsStep(canClone, "Wavelength");
-    } else {
-      absCorProps["CanWorkspace"] = cloneName;
-    }
+    absCorProps["CanWorkspace"] = cloneName;
 
     const bool useCanScale = m_uiForm.ckScaleCan->isChecked();
     if (useCanScale) {
       const double canScaleFactor = m_uiForm.spCanScale->value();
       applyCorrAlg->setProperty("CanScaleFactor", canScaleFactor);
     }
+    const bool useShift = m_uiForm.ckShiftCan->isChecked();
+    if (useShift) {
```

Comment:
If this is the only place `useShift` is used in this method then we can replace:
`if (useShift) {`
with 
`if (m_uiForm.ckShiftCan->isChecked()) {  // If container is shifted`
and remove the line above where `useShift` is defined